docs/nifgen/enums.rst

```diff
@@ -423,7 +423,7 @@ SampleClockSource
 
 
 
-    .. py:attribute:: SampleClockSource.PXI_TRIGGER_LINE_0RTSI_0
+    .. py:attribute:: SampleClockSource.PXI_TRIGGER_LINE_0_RTSI_0
```

Comment:
Both old and new look wrong (old looks more wrong).
Constant in C is `NIFGEN_VAL_RTSI_0`

What should we be doing here?
Do we even want / need this in `nifgen`? It's used for the old enum-based routing API, we should be directing clients towards using string-based routing API.